lib/plugins/aws/lib/naming.js

```diff
@@ -335,6 +335,11 @@ module.exports = {
   getTopicLogicalId(topicName) {
     return `SNSTopic${this.normalizeTopicName(topicName)}`;
   },
+  getTopicDLQPolicyLogicalId(functionName, topicName) {
```

Comment:
I've created this, since the previously generated names for the queue policy looked wrong (ere referencing eventMapping) since they were designed for a Queue and not for a QueuePolicy